Makefile

```diff
@@ -105,10 +107,10 @@ license: .FORCE
 format: .FORCE
```

Comment:
Just FYI, you didn’t start it but you should probably understand the difference between .FORCE/FORCE and .PHONY.

https://www.gnu.org/software/make/manual/html_node/Force-Targets.html

Reply:
Also, the dependency should really be on the binary and the binary target should have a dependency on tools.go so it gets built if the binary doesn’t exist or is older than tools.go.